components/blocks/About.vue

```diff
@@ -131,6 +135,25 @@ export default {
   },
   methods: {
     urlize,
+    async getOpportunities() {
```

Comment:
with nuxt, I think `async fetch` is the idiomatic way to get the data for the page: https://nuxtjs.org/docs/features/data-fetching/

the data structure we get back from workday has the job postings nested somewhere, so we should pull that array out

I think for safety we should double check that the location in each job posting matches `180 George Street`